### PR TITLE
Update webpacker to 5.4.3

### PR DESCRIPTION
**Pull request by jlledom**
Required to merge https://github.com/3scale/porta/pull/3072

### CODE REVIEW

**Comment by jlledom** [comment]
This is not stale, it's waiting for #3054 to be merged first

**Comment by jlledom** [comment]
I merged the new typescript base code

File: package.json [hunk 1]
```diff
@@ -76,9 +76,10 @@
     "@patternfly/react-icons": "^3.14.18",
     "@patternfly/react-styles": "^4.8.2",
     "@patternfly/react-table": "^4.18.14",
-    "@rails/webpacker": "4.3.0",
+    "@rails/webpacker": "^5.4.3",
```
Comment (josemigallas) [inline package.json:79]: Please use fixed versions instead.

File: package.json [hunk 2]
```diff
@@ -76,9 +76,10 @@
...
     "@stripe/react-stripe-js": "^1.10.0",
     "@stripe/stripe-js": "^1.34.0",
+    "babel-loader": "^8.0.6",
```
Comment (josemigallas) [inline package.json:82]: This is also listed under `devDependencies`.

**Review by josemigallas — COMMENTED** [review]
Have you tried to deploy this to preview?

**Comment by jlledom** [comment]
Closing in favor of https://github.com/3scale/porta/pull/3199

**Comment by josemigallas** [comment]
Let's merge this @jlledom

**Review by josemigallas — DISMISSED** [review]

**Comment by akostadinov** [comment]
After migration, no??

**Comment by jlledom** [comment]
we can merge it but not deploy it after migratin, legacy won't accept this

**Comment by akostadinov** [comment]
Then as we agreed, we wont merge things that legacy will not accept before we get rid of it.

**Review by akostadinov — APPROVED** [review]

**Pull request merged**